README.md

```diff
@@ -688,300 +694,369 @@ the following essential section should be provided in user template
-+ `pert_xz` is the perturbation through xz direction used to compute surface energy.
-+ `max_miller` (integer) is the maximum miller index
-+ `static-opt`:(boolean) whether to use atomic relaxation to compute surface energy. if false, the structure will be relaxed.
-+ `relax_box`:(boolean) set true if the box is relaxed, otherwise only relax atom positions.
 
-|Key  | Type  | Example | Discription  |
-| :---------------- | :--------------------- | :-------------------------------------- | :-------------------------------------------------------------|
-| min_slab_size| real number| 10 |  the minimum size of slab thickness |
-|min_vacuum_size | real number| 11 |  the minimum size of  the vacuume width |
-|pert_xz  | real number| 0.01 |  the perturbation through xz direction used to compute surface energy |
-|max_miller  | integer| 2 |  the maximum miller index |
-|static-opt|boolean| false | whether to use atomic relaxation to compute surface energy. if false, the structure will be relaxed. |
-|relax_box | boolean | false | set true if the box is relaxed, otherwise only relax atom positions |
-
-test result
-```
-conf_dir:confs/Al/std-fcc
-Miller_Indices:         Surf_E(J/m^2) EpA(eV) equi_EpA(eV)
-struct-000-m1.1.1m:        0.673     -3.628   -3.747
-struct-001-m2.2.1m:        0.917     -3.592   -3.747
-```
-| Field  | Type | Example| Discription|
-| :---------------- | :--------------------- | :-------------------------------------- | :-------------------------------------------------------------|
-|Miller_Indices| string | struct-000-m1.1.1m | Miller Indices|
-|Surf_E(J/m^2)| real number | 0.673 | the surface formation energy |
-| EpA(eV) | real number | -3.628 | potential energy of the surface configuration |
-| equi_EpA | real number | -3.747 | potential energy of the equilibrium state|
+`reproduce` denotes whether to do `reproduce` or not and the default value is False. 
+
+`init_data_path` is the path of VASP or LAMMPS initial data to be reproduced. `init_from_suffix` is the suffix of the initial data and the default value is "00". In this case, the VASP Interstitial results are stored in `../vasp/confs/std-*/interstitial_00` and the reproduced Interstitial results would be in `deepmd/confs/std-*/interstitial_reprod`. 
+
+`reprod_last_frame` denotes if only the last frame is used in reproduce. The default value is True for eos and surface, but is False for vacancy and interstitial.
+
+Three steps for reproduce calculations are also `make`, `run`, and `post`.
 
-### The content of the auto_test
-To know what actually will dpgen autotest do, including the lammps and vasp script, the input file and atom configuration file auto_test will generate, please refer to https://hackmd.io/@yeql5ephQLaGJGgFgpvIDw/rJY1FO92B
+### More Information of Autotest
+For more information of autotest, please refer to [dpgen wiki](https://github.com/deepmodeling/dpgen/wiki).
```

Comment:
> This PR intends to replace the autotest doc in the wiki. All information in the wiki is supposed to be moved to README

But the wiki is cited here. Should we keep the wiki separate or move it to the documentation site?

Reply:
@kevinwenminion plz move it to the documentation site

Reply:
https://github.com/deepmodeling/dpgen/pull/891 this pr move the wiki to the documentation site

Reply:
I have removed the citation of wiki in README. Users can learn how to use autotest very quickly by reading the updated autotest part in README.